src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -1489,7 +1495,7 @@ export class PostgresStorageAdapter {
       }
     }
 
-    const qs = `SELECT ${columns} FROM $1:name ${wherePattern} ${sortPattern} ${limitPattern} ${skipPattern} ${groupPattern}`;
+    const qs = `SELECT ${columns.join(',')} FROM $1:name ${wherePattern} ${sortPattern} ${limitPattern} ${skipPattern} ${groupPattern}`;
```

Comment:
Wait, how did the tests not catch this? Seems like this would have caused a noticeable problem.

Reply:
yeah that's very very odd. @dplewis ?

Reply:
Good catch. There isn't a test for `project` with multiple fields. This should fix that issue.